### DISCUSSION TITLE
Ender 3 V2 upgrading to 500mm on the Z-axis

### DISCUSSION THREAD

**Discussion by sfavilla2021** [Jyers/Marlin#774] [Q&A] [answered]
Hey all; I'm pretty new to all this; but luv it.
My issue is I want to upgrade the Z height to 500mm; I already bought the hardware.

I am using Jyers V 1.3.1 firmware on Default 4.2.2 board
How can I simply update the height change from 250 to 500?
TIA

**Comment by dkustec** [comment, marked as answer]
You should edit config then compile by yourself.
Changes are there: (Config.h) 
![image](https://user-images.githubusercontent.com/7397515/117721634-662dcb80-b1e0-11eb-93cd-85bfa513e5e5.png)

**Reply by sfavilla2021** [reply]
TY; Any way I can just upload those changes and nothing else witin the config.h file?

**Reply by Jyers** [reply]
Depends what you mean by "upload"

**Reply by msdossajo** [reply]
> TY; Any way I can just upload those changes and nothing else witin the config.h file?

Probably I misunderstood your question.
No, there isn't an easy way of doing it. You will have to compile a new firmware with the new Z height limite.

**Reply by sfavilla2021** [reply]
Thanks all; I'll give it a shot and see what happens.

**Comment by smooreace** [comment]
Its literally editing a text file! Put some effort into bettering yourself
and learn something!

**Reply by sfavilla2021** [reply]
I'm a disabled combat vet; I've learned a lot in my life; like not being a dick!!

**Comment by smooreace** [comment]
Congrats... but clearly not!! But the truth is just that! You can clearly
still type, so stop making excuses.

**Comment by smooreace** [comment]
Just for the record, I am too! But I would NEVER pretend that it was OK to
use it as an excuse or bargaining chip if I wanted something!

**Comment by Tuncay-Ayhan** [comment]
Can I join the p!ssing contest @smooreace ?

**Comment by sfavilla2021** [comment]
Hello again; I am looking at the new Jyers and wanted to update the screen DWin_set to use the new custom icons; how do I make sure that I have the right icon pack from the three available?

**Comment by Cal-LF** [comment]
@sfavilla2021 The pack to use is the dwin-set gotcha. They are the better icons.
Follow display update instructions.

**Comment by sfavilla2021** [comment]
TY, I am trying to find the instructions; where are they located?

**Reply by Cal-LF** [reply]
After downloading the source zip file, see instructions in the display firmware folder.

**Comment by sfavilla2021** [comment]
Ok. Found the instructions. Had to have notebook app set to read the readme.md file.
Do I need to do anything with the icon pack folder?

Scott Favilla
Disabled Veteran
US Army

On Feb 19, 2022, at 7:22 AM, Charles Longfellow ***@***.***> wrote:

﻿

After downloading the source zip file, see instructions in the display firmware folder.

**Reply by BlueWings172** [reply]
@sfavilla2021 

Follow the steps in [this post](https://github.com/Jyers/Marlin/discussions/1768#discussioncomment-1943636) to update the display firmware with the new icons.

**Comment by sfavilla2021** [comment]
Hey all. I tried to upgrade it. Did not change the icons. But now another problem has shown up.
I upgraded to the latest firmware and now my prints will randomly pause mid print and I won’t see it and the print is ruined. I’m using board 4.2.2.

Scott Favilla
Disabled Veteran
US Army

On Feb 19, 2022, at 1:06 PM, BlueWings172 ***@***.***> wrote:

﻿

@sfavilla2021<https://github.com/sfavilla2021>

Follow the steps in this post<https://github.com/Jyers/Marlin/discussions/1768#discussioncomment-1943636> to update you to update the screen firmware with the new icons.
